Dockerfile

```diff
@@ -1,15 +1,22 @@
 FROM docker.io/golang:alpine as builder
 
 ARG YGG_VERSION=v0.4.7
+ENV CGO_ENABLED=0
 
 WORKDIR /src
 
-ENV CGO_ENABLED=0
-
 RUN apk add git
+
 RUN git clone --depth 1 --branch $YGG_VERSION https://github.com/yggdrasil-network/yggdrasil-go.git .
 RUN ./build && go build -o /src/genkeys cmd/genkeys/main.go
 
+RUN git clone https://github.com/threefoldtech/tfgridclient_proxy && cd tfgridclient_proxy/cmds/proxy_server &&\
```

Comment:
Why git clone the repo you are in? Just ADD'ing the sources is better since you can build from local sources or a branch.